about.html

```diff
@@ -0,0 +1,95 @@
+                <li><a href="#campaign" id="campaign">FWC Campaign</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="hero" class="hero-about">
+            <div id="hero-content">
+                <h1>
+                    <span>"Hello! Chess wunderkinds" <br></span>
+                    <span class="main">FIDE WORLD CHESS TEEN TOURNAMENT 2023</span>
+                </h1>
+                <p class="info-about">
+                    The International Chess Federation or World Chess Federation, commonly referred to by its French acronym FIDE, is an international organization based in Switzerland that connects the various national chess federations and acts as the governing body of international chess competition. FIDE was founded in Paris, France, on July 20, 1924.
+                </p>
+                <h2 class="contact-us">
+                    <span>Please contact us per Email for any further questions about the FWC Teen tournament 2023!</span>
+                    <span class="mail-contact">fwc2023@fidee.org</span>   
+                </h2>
+                
+            </div>
+  
+        </section>
+        <section id="logo">
+            <h3 class="section-header">FIDE World Chess Teen Tournament 2023 Logo</h3>
+            <hr>
+            <p class="text-b">The logo of the FWC Teen Tournament 2023 was decided through the logo competition from 2. February to 3. March. </p>
+            <div class="logo-unveil">
+                <img id="fide-logo" src="images/logo-fide.svg" alt="FIDE logo">
+                <div class="logo-text">
+                    <span>FWC Teen Tournament 2023 <br></span>
+                    <span>25th - 28th April 2023 | Astana, Kazakhstan</span>
+                </div>
+            </div>
+        </section>
+        <section id="past">
+            <h3 class="section-header">See the past FWC Teen Tournaments</h3>
+            <hr>
+            <p class="text-b">Take a look at the last two FWC Teen Tournaments which took place in Dubai and Moscow.</p>
+            <ul id="past-list">
+                <li class="past-item1">
+                        <h3>2022</h3>
+                        <p>FWC Teen Tournament, Dubai</p>
+                </li>
+                <li class="past-item2">
+                        <h3>2022</h3>
+                        <p>FWC Teen Tournament, Moscow</p>
+                </li>
+            </ul>
+        </section>
+    </main>
+    <footer>
+        <section id="partners">
+            <h3 class="section-header">Partners</h3>
+            <hr>
+            <div class="partner-logos">
+                <i class="fa-brands fa-google-pay" ></i>
+                <i class="fa-brands fa-old-republic" ></i>
+                <i class="fa-brands fa-d-and-d-beyond" ></i>
+                <i class="fa-brands fa-free-code-camp" ></i>
+                <i class="fa-brands fa-autoprefixer"></i>
+            </div>
+        </section>
```

Comment:
- [ ] Kindly, make sure the partner section is inside the home page and not the About page. you also need to swap the footer of the home page and the About page. The footer of the home page should be white (as the partner section is dark) and the footer of the About page should be dark.

**Your project home page: should have a partner section and white footer**
![image](https://user-images.githubusercontent.com/74411135/235321280-73b9d45a-9d8b-484d-819c-f26fdbd863a5.png)

**Provided design home page: have partner section and white footer**
![image](https://user-images.githubusercontent.com/74411135/235321341-bb29f8a8-d143-4f1b-90a0-0fdd7cf2f78d.png)

**Your project about page: should not have a partner section and should have dark footer**
![image](https://user-images.githubusercontent.com/74411135/235321384-9eb9a1ed-0d89-46c4-8059-f1e2536c3bee.png)

**provided design footer: have no partner section and have dark footer**
![image](https://user-images.githubusercontent.com/74411135/235321411-d85d89b9-da33-4897-912d-229c0e7a773e.png)

This will make your project prettier and make the user know you're an awesome developer as you can precisely follow a professional web developer's design 😃 💪